examples/fft_vs_geom_movie.py

```diff
@@ -0,0 +1,348 @@
+                    if j == 0:
+                        continue
+                    ztext[j-1].set_text("Z{:d} = {:5.3f}".format(j+1, ab))
+
+                # Calculate simple estimate of ellipticity
+                mom_fft = simple_moments(fft_img)
+                mom_phot = simple_moments(geom_img)
+                e_fft = ellip(mom_fft)
+                e_phot = ellip(mom_phot)
+
+                Is = ("$I_x$={:6.3f}, $I_y$={:6.3f}, $I_{{xx}}$={:6.3f},"
+                      " $I_{{yy}}$={:6.3f}, $I_{{xy}}$={:6.3f}")
+                I_fft.set_text(Is.format(mom_fft['Ix'], mom_fft['Iy'],
+                                         mom_fft['Ixx'], mom_fft['Iyy'], mom_fft['Ixy']))
+                I_phot.set_text(Is.format(mom_phot['Ix'], mom_phot['Iy'],
+                                          mom_phot['Ixx'], mom_phot['Iyy'], mom_phot['Ixy']))
+                etext_fft.set_text("$e_1$={:6.3f}, $e_2$={:6.3f}, $r^2$={:6.3f}".format(
+                                   e_fft['e1'], e_fft['e2'], e_fft['rsqr']))
+                etext_phot.set_text("$e_1$={:6.3f}, $e_2$={:6.3f}, $r^2$={:6.3f}".format(
+                                    e_phot['e1'], e_phot['e2'], e_phot['rsqr']))
+
+
+                fft_mom[i] = (mom_fft['Ix'], mom_fft['Iy'],
+                              mom_fft['Ixx'], mom_fft['Iyy'], mom_fft['Ixy'],
+                              e_fft['e1'], e_fft['e2'], e_fft['rsqr'])
+
+                geom_mom[i] = (mom_phot['Ix'], mom_phot['Iy'],
+                              mom_phot['Ixx'], mom_phot['Iyy'], mom_phot['Ixy'],
+                              e_phot['e1'], e_phot['e2'], e_phot['rsqr'])
+
+                writer.grab_frame(facecolor=fig.get_facecolor())
+
+                bar.update()
+
+    def symmetrize_axis(ax):
+        xlim = ax.get_xlim()
+        ylim = ax.get_ylim()
+        lim = min(xlim[0], ylim[0]), max(xlim[1], ylim[1])
+        ax.set_xlim(lim)
+        ax.set_ylim(lim)
+        ax.plot(lim, lim)
+
+    # Centroid plot
+    fig, axes = plt.subplots(1, 2, figsize=(10, 6))
+    axes[0].scatter(fft_mom[:, 0], geom_mom[:, 0])
+    axes[1].scatter(fft_mom[:, 1], geom_mom[:, 1])
+    axes[0].set_title("Ix")
+    axes[1].set_title("Iy")
+    for ax in axes:
+        ax.set_xlabel("Fourier Optics")
+        ax.set_ylabel("Geometric Optics")
+        symmetrize_axis(ax)
+    fig.tight_layout()
+    fig.savefig(args.out+"centroid.png", dpi=300)
+
+    # Second moment plot
+    fig, axes = plt.subplots(1, 3, figsize=(16, 6))
+    axes[0].scatter(fft_mom[:, 2], geom_mom[:, 2])
+    axes[1].scatter(fft_mom[:, 3], geom_mom[:, 3])
+    axes[2].scatter(fft_mom[:, 4], geom_mom[:, 4])
+    axes[0].set_title("Ixx")
+    axes[1].set_title("Iyy")
+    axes[2].set_title("Ixy")
+    for ax in axes:
+        ax.set_xlabel("Fourier Optics")
+        ax.set_ylabel("Geometric Optics")
+        symmetrize_axis(ax)
+    fig.tight_layout()
+    fig.savefig(args.out+"2ndMoment.png", dpi=300)
```

Comment:
> Do you know how to estimate the size of the missing isotropic kernel?

Roughly, I can play with this in Mathematica a bit to see if I can get a size estimate.  The details are that the second kick PSF is the Fourier transform of the exponential of an integral over the turbulence power spectrum.  I suspect we'll have to add a new `GSObject` for this, though maybe it'll be a private one since I don't see how it would be useful outside of the geometric optics approximation.